### ISSUE TITLE
Add reward on death / kill / zombi etc ...

### ISSUE THREAD

**Issue by Riddler76** [uEssentials/uEssentials#323]
Hey,

i have a suggestion for you :)

Add reward on differents event : 

- Kill
- Death
- Zombie
- Animal
....

Rewards : 

- Economy (Credit uConomy)
- Experience
- Kits ? Item ? Command ?

**Comment by viperp12** [comment]
Is it possible to add to this idea, specifically xp rewards for defeating a player? What would make this absolutely amazing for a PVP server would be if we can keep track of how long that player has been alive, in game, without dying, and uptick the xp value, possibly to a max.

Been thinking about how to do this for a little while. Lastly, and less important, but still fundamentally a fun concept, would be to add a way for players to see how valuable other players are. :)

**Comment by Riddler76** [comment]
The base idea is reward player when kill another player. Reward it with XP.

Other idea are good, i give you many suggestions but are just suggestions ;)

Reward for play time is also a nice idea !

**Comment by viperp12** [comment]
Definitely, I think it would give players incentive to pvp and have fun doing it.

**Comment by viperp12** [comment]
Any chance on this becoming a reality?
